Mathlib/Analysis/SpecialFunctions/Log/Base.lean

```diff
@@ -64,11 +64,18 @@ lemma logb_self_eq_one_iff : logb b b = 1 ↔ b ≠ 0 ∧ b ≠ 1 ∧ b ≠ -1 :
   Iff.trans ⟨fun h h' => by simp [logb, h'] at h, div_self⟩ log_ne_zero
 
 @[simp]
-theorem logb_abs (x : ℝ) : logb b |x| = logb b x := by rw [logb, logb, log_abs]
+theorem logb_abs_base (b x : ℝ) : logb |b| x = logb b x := by rw [logb, logb, log_abs]
```

Comment:
For reviewers: 
This isn't actually a rename: the lemma `logb_abs` is unchanged, the lemma `logb_abs_base` is added and github shows a confusing diff.